engine/physics/src/box2d/Box2D/Dynamics/b2World.cpp

```diff
@@ -911,7 +911,8 @@ void b2World::Step(float32 dt, int32 velocityIterations, int32 positionIteration
 	b2TimeStep step;
 	step.dt = dt;
 	step.velocityIterations	= velocityIterations;
-	step.positionIterations = positionIterations;
+	step.positionIterations = positionIterations; 
```

Comment:
We want an as clean copy of Box2D as possible. Please revert this change.

Reply:
Ok, so no touching to b2World at all